hw/ip/kmac/dv/dpi/vendor/kerukuro_digestpp/detail/functions.hpp

```diff
@@ -89,4 +89,5 @@ inline void zero_memory(std::string &s) {
 }  // namespace detail
 }  // namespace digestpp
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
Looks like your editor/git is configured to modify some of the vendored in files? That's probably what causes CI to fail.

Reply:
Should be fixed now.